packages/runner-ct/src/plugins/README.md

```diff
@@ -0,0 +1 @@
+This is a temporal folder. Everything here should be an external dependency and should be loaded asynchronously. Until we design the front-end plugins system use some out-of-the box UI extensions with lazy-loading.
```

Comment:
will this work properly when the runner-ct is packaged inside the binary?
I have no idea how the binary is published yet.

Reply:
This depends on how it is built :D 
In theory there should not be any problems cause it is the same as we are doing for serving the main file of runner ct, but with a little bit different path. But who knows